client/state/analytics/actions.js

```diff
@@ -24,7 +24,7 @@ const joinAnalytics = ( analytics, action ) =>
 export const composeAnalytics = ( ...analytics ) => ( {
 	type: ANALYTICS_MULTI_TRACK,
 	meta: {
-		analytics: analytics.map( property( 'meta.analytics' ) )
+		analytics: analytics.map( property( 'meta.analytics[0]' ) )
```

Comment:
FYI opened a PR about it, complete with a test that would fail without my fix: https://github.com/Automattic/wp-calypso/pull/10923

It's quite the wall of text, but hopefully it's clear what's going on there.